app/assets/stylesheets/administrate/application.scss

```diff
@@ -12,3 +11,14 @@
 @import "components/cells";
 @import "components/pagination";
 @import "components/search";
+
+.sr-only {
```

Comment:
note: this class ironically was coming from `datetime_picker`, which `administrate` no longer uses

I've just inlined the specific class since it's all we need to preserve the existing behaviour